src/Settings.php

```diff
@@ -117,7 +117,7 @@ public function get( $setting ) {
 		$setting = str_replace( 'siwk_', '', $setting );
 		if ( 'scope' === $setting ) {
 			// These scopes are required for full functionality and shouldn't be modified by the merchant, and must be excluded from the filter.
-			$required = 'openid offline_access customer:login profile:name profile:email profile:phone profile:billing_address';
+			$required = 'openid offline_access customer:login profile:name profile:email profile:phone profile:billing_address ';
```

Comment:
When we return this value, we should probably add a trim to remove any trailing whitespace characters incase none of the optional scopes are added, just to not cause an errors when sending this to Klarna.